ui/app/templates/variables/variable.hbs

```diff
@@ -0,0 +1,12 @@
+{{page-title "Secure Variables: " this.model.path}}
+<Breadcrumb @crumb={{this.breadcrumb}} />
```

Comment:
While testing this manually I noticed the path is all displayed as a single string. Are the paths supposed be presented as breadcrumbs or is that just not wired up yet? (Maybe that's part of https://github.com/hashicorp/nomad/issues/13094?)

Reply:
Yep, in the above issue I'll be splitting paths into trees for better navigation. Like Github and other tree displayers, I'll be handling those non-terminal paths (not linked, single string) different than terminal ones (linked, separate from rest of string)